server/public/model/feature_flags.go

```diff
@@ -49,6 +49,8 @@ type FeatureFlags struct {
 
 	DataRetentionConcurrencyEnabled bool
 
+	AllowDisableWebappPrefetch bool
```

Comment:
Instead of a whole feature flag for this, I think we could actually put this behind `ServiceSettings.EnableClientPerformanceDebugging`. I might have mentioned it already, but it enables the Advanced Settings > Performance Debugging which contains a few less user-friendly settings that test similar things. It seems like a good place to put this since it has a lot of that settings infrastructure already set up

Reply:
Good call, made that change